misc/help.talon

```diff
@@ -1,6 +1,7 @@
 help alphabet: user.help_alphabet(user.get_alphabet())
 help context$: user.help_context()
 help active$: user.help_context_enabled()
+help search <phrase>$: user.help_search(phrase)
```

Comment:
I'd vote to user user.text here instead of phrase.

Reply:
@knausj85 Updated

Reply:
I think the ideal thing is to actually use a matcher consisting of all words in your commands, but I was getting some weird parses when I tried that.

Reply:
Looks like the help.talon thingy didn't get updated? 

`help search <user.text>$: user.help_search(user.text)`

or am I missing something?

Reply:
My bad, didn't add the file. Check again.